src/styles/_extends.scss

```diff
@@ -0,0 +1,3 @@
+%smooth {
+  transition: all $smooth linear 0;
```

Comment:
I mean list all transition properties
for example: `transition: transform $smooth, color $smooth`